opentargets.json

```diff
@@ -52,6 +52,57 @@
       ],
       "additionalProperties": false
     },
+    {
+      "properties": {
+        "datasourceId": {
+          "const": "cancer_genome_interpreter"
+        },
+      "biomarker": {
+        "$ref": "#/definitions/biomarker"
+      },
+      "datatypeId": {
+        "$ref": "#/definitions/datatypeId"
+      },
+      "diseaseFromSource": {
+        "$ref": "#/definitions/diseaseFromSource"
+      },
+      "diseaseFromSourceMappedId": {
+        "$ref": "#/definitions/diseaseFromSourceMappedId"
+      },
+      "drugFromSource": {
+        "$ref": "#/definitions/drugFromSource"
+      },
+      "drugId": {
+        "$ref": "#/definitions/drugId"
+      },
+      "drugResponse": {
+        "$ref": "#/definitions/drugResponse"
+      },
+      "confidence": {
+        "$ref": "#/definitions/confidence"
+      },
```

Comment:
This is a nitpick, but I think the properties should be sorted by name, as done in the other subschemas. So that `confidence` comes after `biomarker`, etc.